Dockerfile.dev

```diff
@@ -12,6 +12,6 @@ RUN pip install poetry
 WORKDIR /app
 ENV PYTHONPATH /app
 
-COPY pyproject.toml poetry.lock .
+COPY pyproject.toml poetry.lock README.md .
```

Comment:
Is this necessary for the install?

Reply:
I saw an error message from poetry complaining that this file was missing, but adding it makes it even more unhappy. I think not including the source for the app before the `poetry install` in the Dockerfile is making poetry 😒

Reply:
Can you paste the error? I wasn't having any issues with this